package.json

```diff
@@ -13,8 +13,8 @@
   ],
   "scripts": {
     "test": "ts-mocha 'test/unit/*.[jt]s'",
-    "lint": "tsc -b && eslint '{lib,test}/**/*.{t,j}s'",
-    "lint:fix": "eslint --fix lib/ test/",
+    "lint": "eslint '{lib,test}/**/*.{t,j}s'",
+    "lint:fix": "npm run lint -- --fix",
     "build": "npm run typings && rollup -c --compact",
     "build:debug": "npm run build -- --sourcemap",
     "typings": "tsc -b typings.tsconfig.json && mv lib/types.d.ts typings.d.ts && cat module-declaration.d.ts >> typings.d.ts",
```

Comment:
Could we eliminate typings somehow? Doesn't make sense to have them in 100% TS project.

Reply:
I did some testing on this. We'd still have a separate command to generate typings, but using the source files instead of the `types.ts` file. The types would still mostly depend on the `types.ts` file as it is heavily imported throughout the code.
I'm currently making sure that the developer experience, especially autocomplete features, work exactly the same.

Reply:
I think splitting it into two PRs makes sense for this issue. The changes for this can be made irrespective of the rest of the work done in this PR without any overhead, and I imagine it is better to get the changes out quicker so other people can work in the complete typescript context @jelz What do you think?